aten/src/THC/THCTensorRandom.cu

```diff
@@ -37,7 +40,7 @@ THC_API __host__ void THCRandom_getRNGState(at::Generator *gen_, THByteTensor *r
   memcpy(THByteTensor_data(rng_state) + states_size + seed_size, &offset, offset_size);
 }
 
-THC_API __host__ void THCRandom_setRNGState(at::Generator *gen_, THByteTensor *rng_state)
+extern "C" __host__ void THCRandom_setRNGState(at::Generator *gen_, THByteTensor *rng_state)
```

Comment:
https://github.com/pytorch/pytorch/blob/1d0d9b10ea7a4385947402ed0628ae1bace1c059/aten/src/THC/THCGeneral.h.in#L21
https://github.com/pytorch/pytorch/blob/1d0d9b10ea7a4385947402ed0628ae1bace1c059/c10/macros/Export.h#L105
How about c10 export part?

Reply:
It's handled in the header, and that annotation is properly propagated.